_components/customisation.md

```diff
@@ -0,0 +1,199 @@
+
+A good place to start is to ask for feedback in the <a href="https://financialtimes.slack.com/messages/{{site.data.contact.slack}}">#{{site.data.contact.slack}}</a> Slack channel. The core Origami team, along with the design team, can help identify similar existing styles or help make any relevant component updates to support your usecase without customisation.
+
+## Brands
+
+Most Origami components are branded to provide a distinct appearance within different contexts. A brand may be thought of as a kind of theme, but applies to all components within a project.
+
+### Supported brands
+
+Origami maintained brands include:
+- master: FT branding for public ft.com sites and affiliates.
+- internal: Style suitable for internal products, tools, and documentation.
+- whitelabel: Base, structural styles only to build on and customise.
+
+Taking `o-table` as an example, the "master" brand version offers a "row stripes" feature, which uses "master" brand colours such as "paper" and "wheat":
+{% include demo.html component="o-table" demo="row-stripes" brand="master" height="242px" %}
+
+The "internal" brand version also supports "row stripes" but uses a different colour palette:
+{% include demo.html component="o-table" demo="row-stripes" brand="internal" height="242px" %}
+
+The "whitelabel" brand does not support "row stripes" and therefore outputs no styles to support that feature:
+{% include demo.html component="o-table" demo="basic" brand="whitelabel" height="256px" %}
+
+### Choose A Brand
+
+The "master" brand is the default brand, but your project may use any of the provided Origami brands. The brand is configured once and affects all components used by your project.
+
+To select a brand within your project, see the tutorial for your method of including Origami components:
+
+- [Build service tutorial](/docs/tutorials/build-service/#selecting-a-brand)
+- [Manual build tutorial](/docs/tutorials/manual-build/#selecting-a-brand)
+
+### Customise A Brand
+
+If your project has access to a component's Sass API (see the [manual build tutorial](/docs/tutorials/manual-build/#selecting-a-brand)) it is possible to customise your chosen brand. For example, at the time of writing, the [`oTypographySetFont`](https://registry.origami.ft.com/components/o-typography@6.4.5/sassdoc?brand=master#mixin-otypographysetfont) mixin can be used to customise component fonts. This customises the font used by all components including in your project.
+
+<aside>
+Not all components support brand customisation but the Origami team are happy to add support as needed. Ask the core Origami team for new features in the <a href="https://financialtimes.slack.com/messages/{{site.data.contact.slack}}">#{{site.data.contact.slack}}</a> Slack channel.
```

Comment:
I reworded this aside in a way where (to me at least) it reads more like anyone can add brand customisation support to a component

```suggestion
Not all components support brand customisation but the Origami team are happy for support to be added as needed. Ask the core Origami team about new features in the <a href="https://financialtimes.slack.com/messages/{{site.data.contact.slack}}">#{{site.data.contact.slack}}</a> Slack channel.
```